rules/threat_intel/threat_intel_rapid7_threat_command.toml

```diff
@@ -53,15 +53,9 @@ or a [custom integration](https://www.elastic.co/guide/en/security/current/es-th
 
 More information can be found [here](https://www.elastic.co/guide/en/security/current/es-threat-intel-integrations.html).
 
-## Max Signals
+### Additional notes
 
-This rule is configured to generate more **Max alerts per run** than the default 1000 alerts per run set for all rules. This is to ensure that it captures as many alerts as possible.
-
-**IMPORTANT:** The rule's **Max alerts per run** setting can be superseded by the `xpack.alerting.rules.run.alerts.max` Kibana config setting, which determines the maximum alerts generated by _any_ rule in the Kibana alerting framework. For example, if `xpack.alerting.rules.run.alerts.max` is set to 1000, this rule will still generate no more than 1000 alerts even if its own **Max alerts per run** is set higher.
-
-To make sure this rule can generate as many alerts as it's configured in its own **Max alerts per run** setting, increase the `xpack.alerting.rules.run.alerts.max` system setting accordingly.
-
-**NOTE:** Changing `xpack.alerting.rules.run.alerts.max` is not possible in Serverless projects.
+For information about troubleshooting maximum alerts warning please refer to this [guide](https://www.elastic.co/guide/en/security/current/alerts-ui-monitor.html#troubleshoot-max-alerts).
```

Comment:
I'm not sure if this is needed for this one

Reply:
Maintaining the style of the previous change to also have consistent setup notes on max_signals = 1000

Reply:
I think there is less likelihood that this rule will hit the limit. It's ok to remove this note.